packages/trader/src/App/Containers/NotificationsDialog/notifications-dialog.jsx

```diff
@@ -85,6 +89,24 @@ class NotificationsDialog extends React.Component {
                                                 <div className='notifications-item__message'>
                                                     {item.message}
                                                 </div>
+                                                {!ObjectUtils.isEmptyObject(item.action) &&
+                                                    <React.Fragment>
+                                                        { item.action.route ?
+                                                            <BinaryLink
+                                                                className={classNames('btn', 'btn--secondary', 'notifications-item__cta-button')}
+                                                                to={item.action.route}
+                                                            >
+                                                                <span className='btn__text'>{item.action.text}</span>
+                                                            </BinaryLink>
+                                                            :
+                                                            <Button
+                                                                className={classNames('btn--secondary', 'notifications-item__cta-button')}
+                                                                onClick={item.action.onClick}
+                                                                text={item.action.text}
+                                                            />
+                                                        }
+                                                    </React.Fragment>
+                                                }
```

Comment:
line 89-109 could be a shared `<NotificationsContent />` component and reused here and in `notification.jsx`

Reply:
Thanks, I'll handle this in an upcoming PR for storing the already closed notifications by user.